Doc/library/urllib.rst

```diff
@@ -196,6 +200,10 @@ High-level interface
       the size of the data it has downloaded, and just returns it.  In this case you
       just have to assume that the download was successful.
 
+   .. versionchanged:: 2.7.9
+      The *context* parameter was added. All the neccessary certificate and hostname
```

Comment:
Two spaces after period please. And it looks to me a `versionchanged` note is also lacked in `URLopener`?

Reply:
I fixed the spaces in `urlretrieve` and added a `versionchanged` in `URLopener`. I also fixed the spaces and the wrapping in the `versionchanged` of `urlopen`, which was not consistent to what you suggested.